File: tests/testdata/codegen/nested-invokes-output-versioned-pp/nodejs/nested-invokes-output-versioned.ts
import * as pulumi from "@pulumi/pulumi";
import * as std from "@pulumi/std";

const example = std.replaceOutput({
    text: std.upperOutput({
        input: "hello_world",
    }).apply(invoke => invoke.result),
    search: "_",
    replace: "-",
});

File: tests/testdata/codegen/nested-invokes-output-versioned-pp/go/nested-invokes-output-versioned.go
package main

import (
	"github.com/pulumi/pulumi-std/sdk/go/std"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		_ = std.ReplaceOutput(ctx, std.ReplaceOutputArgs{
			Text: std.UpperOutput(ctx, std.UpperOutputArgs{
				Input: pulumi.String("hello_world"),
			}, nil).ApplyT(func(invoke std.UpperResult) (string, error) {
				return invoke.Result, nil
			}).(pulumi.StringOutput),
			Search:  pulumi.String("_"),
			Replace: pulumi.String("-"),
		}, nil)
		return nil
	})
}
